models/novelGenres.js

```diff
@@ -0,0 +1,12 @@
+const novelGenres = (connection, Sequelize, genres, novels) => {
+  return connection.define('novelsGenres', {
```

Comment:
Decide on pluralization here and be consistent. File is `novelGenres`, function is `novelGenres` but table is `novelsGenres`.